plugins/nanospell/plugin.js

```diff
@@ -313,28 +327,33 @@
 
 			/* #2 setup layer */
 			/* #3 nanospell util layer */
-			var start = function () {
+			function start() {
 				editor.getCommand('nanospell').setState(CKEDITOR.TRISTATE_ON);
 				state = true;
-				var words = getWords(editor.document.$.body, maxRequest);
-				if (words.length == 0) {
-					render();
-				} else {
-					send(words);
-				}
-			};
-			var stop = function () {
+
+				startSpellCheckTimer(50);
+			}
+
+			function stop() {
 				editor.getCommand('nanospell').setState(CKEDITOR.TRISTATE_OFF);
 				state = false;
 				clearAllSpellCheckingSpans(editor.editable());
-			};
+			}
 
 			function checkNow() {
-				if (!selectionCollapsed()) {
+				if (!selectionCollapsed() || self._isBusy) {
 					return;
 				}
 				if (state) {
-					start();
+
```

Comment:
What does `state` actually store? Could we improve it's name? How is it different from `_isBusy`?